src/osx/webrequest_urlsession.mm

```diff
@@ -12,7 +12,7 @@
 
 #include "wx/webrequest.h"
 
-#if wxUSE_WEBREQUEST_URLSESSION
+#if wxUSE_WEBREQUEST && wxUSE_WEBREQUEST_URLSESSION
```

Comment:
I think `wx/chkconf.h` should check that `wxUSE_WEBREQUEST==1` if `wxUSE_WEBREQUEST_URLSESSION==1`.

Reply:
I suppose it is possible to use curl on macOS and disable URLSession, this check would prevent that.

Reply:
Ah, yes, you're right, of course, sorry.